integration_tests/coverage_report/__tests__/sum-test.js

```diff
@@ -15,4 +15,11 @@ describe('sum', () => {
   it('adds numbers', () => {
     expect(sum(1, 2)).toEqual(3);
   });
+  // Required for test coverage of jasmine-check-install.js:
+  check.it('generates numbers', {times: 1}, [gen.posInt], i => {
```

Comment:
is this because jasmine-check-install shows up in the code coverage? I've been meaning to exclude setup files from that list.